### ISSUE TITLE
using React, I am getting error -> TypeError: ky is not a function; @ get imdb.ts:679

### ISSUE THREAD

**Issue by lestgabo** [worr/node-imdb-api#84]
used the example code  
```   
// not using * as imdb gives a different webpack_module error 
import * as imdb from 'imdb-api';

imdb.get({ name: 'The Toxic Avenger' }, { apiKey: EnvironmentVariables.omdbKey, timeout: 30000 }).then(console.log).catch(console.log);
```
I used `yarn add imdb-api` does that have any relevance? Or is this a node/server only api?

**Comment by worr** [comment]
I try and maintain this so that it's not explicitly dependent on any `node` specific modules, but admittedly I'm a bit ignorant of frontend development and common practices there. When you did `yarn add`, did this resolve all the dependencies and add `ky` to your bundle?

**Comment by lestgabo** [comment]
this was in yarn.lock
```
imdb-api@^4.3.0:
  version "4.3.0"
  resolved "https://registry.yarnpkg.com/imdb-api/-/imdb-api-4.3.0.tgz#b0df4aecb9a1cc8338073466c418ddb414513410"
  integrity sha512-MMqDUiXYXDdEmDDl1dO3E/kAI49iCMNfVGjKINyF3L4LU9w7ZjGE7MdahehtYnsY7fOTniSHi4XkdH1UnZHAhA==
  dependencies:
    ky "^0.23.0"
    ky-universal "^0.8.2"
```
but nothing in in package.json, so I tried to manually install them `yarn add ky ky-universal`
now I have this inside `package.json`
```
"dependencies": {
         ...
        "ky": "^0.24.0",
        "ky-universal": "^0.8.2",
}
```

but I still got the same error hahahhaha too bad.
```
TypeError: ky is not a function
    get imdb.ts:679
    get imdb.ts:569
```

**Comment by lestgabo** [comment]
I am just going to use the omdb api 😭😭😭😭 I really wanted to use your api since its up to date and you are active

**Comment by worr** [comment]
Sorry mine didn't work out. I'll open a separate issue about this so that it's tracked and I can verify that it works in frontends in the future. I'll keep it separate so you don't get tons of notifications while I work on it.

**Comment by worr** [comment]
Fixed by 4e4f69679835c4a2f01698b7c87410acfd37ebd2
